code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm

```diff
@@ -193,3 +193,31 @@
 		pulling = null
 		usr.pulledby = null
 	..()
+		
+/obj/item/wheelchair
+	name = "wheelchair"
+	desc = "A folded wheelchair that can be carried around."
+	icon = 'icons/obj/furniture.dmi'
+	icon_state = "wheelchair_folded"
+	item_state = "wheelchair"
+	w_class = ITEMSIZE_HUGE // Can't be put in backpacks. Oh well.
+
+/obj/item/wheelchair/attack_self(mob/user)
+		var/obj/structure/bed/chair/wheelchair/R = new /obj/structure/bed/chair/wheelchair(user.loc)
+		R.add_fingerprint(user)
+		R.name = src.name
```

Comment:
You should add R.desc = src.desc below here so the description will carry over when folding and unfolding